src/devices/cpu/zezinho/zezinho.cpp

```diff
@@ -0,0 +1,123 @@
+// license:GPL-2.0+
+// copyright-holders:Felipe Sanches; Werner Moecke
+
```

Comment:
Beyond the obvious conflict and the necessary update to latest, I wonder here if by GPL-2.0+ you really meant LGPL instead.

Reply:
I frequently adopt the GPL for my drivers.

And I use whatever license was already adopted on pieces of code written by others when I contribute changes to the core. (But I do that much less often than working on drivers)